.gitlab-ci.yml

```diff
@@ -97,6 +95,23 @@ publish-docker:
   script:
     - scripts/gitlab/publish-docker.sh parity
 
+publish-snap:
+  stage:                           publish
+  only:                            *releaseable_branches
+  image:                           parity/snapcraft:gitlab-ci
+  variables:
+    BUILD_ARCH:                  amd64
```

Comment:
This will only be amd64, right?
Any thoughts on snaps for other architectures?

Reply:
Which ones did you have in mind?

Reply:
In here we are building all the possible ones, just because it's easy in that case:
https://github.com/elopio/parity-snap/blob/2b7006abad59f2ff6af211bf021bf8dd072ffece/.travis.yml#L14

Intuitively, I think arm is important. But the snap store doesn't show me metrics about that to confirm it. They might have some number accessible through the backend, I'll ask.